lib/auth/webauthnwin/api.go

```diff
@@ -158,6 +161,19 @@ func Register(
 	}, nil
 }
 
+var (
+	// promptPlatformMessage is the message shown before Touch ID prompts.
+	promptPlatformMessage = "Using platform authenticator, follow the OS dialogs"
+	// promptWriter is the writer used for prompt messages.
+	promptWriter io.Writer = os.Stderr
```

Comment:
@codingllama I saw that touchid prompt is using os.Stderr instead of os.Stdout for prompt, and reason for that?

Reply:
I think that's what lib/client does, but to be fair I don't think `tsh` is totally consistent with stdout/stderr usage.